### DISCUSSION TITLE
Can werkzeug use ASGI ?

### DISCUSSION THREAD

**Discussion by t-el** [pallets/werkzeug#2523] [Q&A]
can ASGI be in future released

**Comment by ThiefMaster** [comment]
If you need ASGI, use [Quart](https://github.com/pallets/quart/). Its API is almost identical to that of Flask.

**Reply by t-el** [reply]
I know about it , but I am using werkzug as module for my project

**Reply by ThiefMaster** [reply]
Werkzeug is for WSGI, not ASGI ;)

**Reply by t-el** [reply]
Ok thank you
